src/Package/Impl/ExportDialog/EnumValueConverter.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace Microsoft.VisualStudio.R.Package.ExportDialog {
+    public abstract class EnumValueConverter<T> : IValueConverter {
```

Comment:
WPF ComboBox can be operated in two modes:
- If list of items is fixed and they are just string key/value pairs, they can be specified directly in XAML:
```xml
<ComboBox>
    <ComboBoxItem Content="components:Resources.PageProperties_Portrait" Tag="portrait" IsSelected="True" />
    <ComboBoxItem Content="components:Resources.PageProperties_Landspace" Tag="landspace" />
</ComboBox>
```
- If list of items isn't fixed or requires a specific ViewModel for its item, you will use `SelectedValuePath` and `DisplayMemberPath` properties.

So this converter isn't required. It also has an implementation overkill with two separate dictionaries, so I'd rather not add it to the repo even for the future.